src/containerapp/azext_containerapp/containerapp_env_decorator.py

```diff
@@ -2,12 +2,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-from azure.cli.command_modules.containerapp._utils import get_default_workload_profiles, safe_set
+from azure.cli.command_modules.containerapp._utils import get_default_workload_profiles, safe_set, _ensure_identity_resource_id
 from knack.log import get_logger
 
 from azure.cli.command_modules.containerapp.containerapp_env_decorator import ContainerAppEnvCreateDecorator, \
     ContainerAppEnvUpdateDecorator
+from azure.cli.command_modules.containerapp._models import ManagedServiceIdentity
```

Comment:
why do you need to import from azure.cli.command_modules.containerapp._models? why not add in extension to avoid dependency of core cli?

Reply:
updated